src/common/fetch/download-json.injectable.ts

```diff
@@ -4,15 +4,14 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import type { RequestInit, Response } from "node-fetch";
-import type { JsonValue } from "type-fest";
 import type { AsyncResult } from "../utils/async-result";
 import fetchInjectable from "./fetch.injectable";
 
 export interface DownloadJsonOptions {
   signal?: AbortSignal | null | undefined;
 }
 
-export type DownloadJson = (url: string, opts?: DownloadJsonOptions) => Promise<AsyncResult<JsonValue, string>>;
+export type DownloadJson = (url: string, opts?: DownloadJsonOptions) => Promise<AsyncResult<unknown, string>>;
```

Comment:
Why `unknown` instead of `JsonValue`?

Reply:
Because due to typescript limitations `JsonValue` is actually a rather useless type. Like `unknown` you have to constantly runtime check (or cast) the fields you are dealing with. Furthermore, if you just don't care about the type safety you have to cast to `unknown` first before casting to some other interface because `tsc` complains about "no overlap".